src/App.js

```diff
@@ -1,5 +1,4 @@
 import './App.css';
-// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] [ OPTIONAL] it would be better to remove all the boilerplate files that came from create-react-app like `app.test.js` `setup...`